k/kafka-go/kafka-go_ubi_8.5.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package       : kafka-go
+# Version       : v0.2.0
+# Source repo   : https://github.com/segmentio/kafka-go.git
+# Tested on     : UBI: 8.5
+# Language      : GO
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Ambuj Kumar <Ambuj.Kumar3@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+
+PACKAGE_NAME=kafka-go
+PACKAGE_VERSION=${1:-v0.2.0}
+PACKAGE_URL=https://github.com/segmentio/kafka-go.git
+
+yum install go git -y
+
+export GOPATH=/home/tester/go
+OS_NAME=$(grep ^PRETTY_NAME /etc/os-release | cut -d= -f2)
+
+mkdir -p $GOPATH/src/github.com/segmentio && cd $GOPATH/src/github.com/segmentio
+rm -rf $PACKAGE_URL
+git clone $PACKAGE_URL
+cd $PACKAGE_NAME
+git checkout $PACKAGE_VERSION
+go get -u golang.org/x/lint/golint
+cp /home/tester/go/bin/golint /usr/bin
+
+go mod init
```

Comment:
do we need error handling for this and the following statements?

Reply:
required changes done